_faq/about/03-costs.md

```diff
@@ -6,8 +6,12 @@ tags:
 ---
 ## Registration
 
-Registration for each event will be via a simple Google Form owned by Soc-RSE. All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors. 
+Registration for each event will be via dedicated registration link on its webpage.
+All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
```

Comment:
```suggestion
All data will be deleted from Indico after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
```

Please note, that we cannot promise deletion of all information as we cannot guarantee what happens to the data that was forwarded to sponsors.